sandbox/settings.py

```diff
@@ -254,9 +254,6 @@ class Base(Configuration):
     MACHINA_MARKUP_WIDGET = "ashley.editor.widgets.DraftEditor"
     MACHINA_PROFILE_AVATARS_ENABLED = False
     MACHINA_USER_DISPLAY_NAME_METHOD = "get_public_username_with_default"
-    MIGRATION_MODULES = {
```

Comment:
Ah! I was wondering why you added this, now I'm wondering why you're removing it. Can you explain? :stuck_out_tongue:

Reply:
no problem ! but do you know I'm not finished yet (lol I'm in wip), I actually have lint problem with my overload that I've just managed to fix, it's not released yet ! Anyway it's always a pleasure to exchange so It doesn't matter !
To answer, this was my first choice on overloading the forum_conversation app. If you don't want to copy all the migrations files you can set this option in the settings. I didn't like importing all the files knowing I had done no change in the model, I thought it could be confusing so using this option allowed that. 
For this commit, as I'm changing the model on adding index, I can't use it anymore and I have to copy all the migration files.

Reply:
@sampaccoud For reference, you'll find explanations [here](https://django-machina.readthedocs.io/en/stable/customization/overriding_applications.html#import-the-application-models-if-needed) about the 2 options available concerning the migration files, when overriding a django-machina app.